MANIFEST.in

```diff
@@ -1,2 +1,4 @@
 recursive-include energyplus_regressions *.config
 recursive-include energyplus_regressions *.png
+recursive-include energyplus_regressions *.ico
+recursive-include energyplus_regressions *.icns
```

Comment:
Now that we have icons for all platforms, we should package them up during wheel creation.